### PR TITLE
Changed two files

### PR DESCRIPTION
**Pull request by dpjohnston**

### CODE REVIEW

**Review by cotycrosby — CHANGES_REQUESTED** [review]
Hey I don't like using p tags on the index file

**Comment by dpjohnston** [comment]
> Hey I don't like using p tags on the index file

Why the hell not?

File: header.php [hunk 1]
```diff
@@ -33,5 +33,6 @@
       <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
     </form>
   </div>
+  <p>Who cares?</p>
```
Comment (cotycrosby) [inline header.php:36]: Why would you put this in?

**Review by cotycrosby — CHANGES_REQUESTED** [review]
Curious about header file

File: header.php (hunk 1 above)
Comment (dpjohnston) [reply header.php:36]: Who cares?

**Pull request merged**